src/InputStream.c

```diff
@@ -117,42 +117,45 @@ static int encryptData(const unsigned char* plaintext, int plaintextLen,
     int len;
     
     if (AppVersionQuad[0] >= 7) {
-        EVP_CIPHER_CTX_init(&cipherContext);
+        if ((cipherContext = EVP_CIPHER_CTX_new()) == NULL) {
```

Comment:
Let's use EVP_CIPHER_CTX_reset so we don't have to reallocate a context for every input packet.